handbook/business-operations/README.md

```diff
@@ -320,6 +320,40 @@ Within 60 days of the end of the year, follow these steps:
 - Once cancelled, update the recurring expenses section of [The Numbers](https://docs.google.com/spreadsheets/d/1X-brkmUK7_Rgp7aq42drNcUg8ZipzEiS153uKZSabWc/edit#gid=2112277278) to reflect the cancellation by changing the projected monthly burn in column G to $0 and adding "CANCELLED" in front of the vendor's name in column C.
 
+### Review an NDA
+We need to review an NDA anytime a vendor, customer or other party wants to:
+1. Use their own NDA rather than Fleet's standard NDA, or
+2. "Redline" (modify) Fleet's NDA by removing, adding or altering its terms.
+
+We should always seek to use Fleet's own NDA first, without alteration. 
+
+When reading an NDA, we want to pay close attention to the following:
+1. We want to be sure that the confidentiality obligations of the NDA are reciprocal.  Fleet and the other party to the agreement should be bound to the same standards of confidentiality toward the handling of each other's confidential information.
+2. _Fleet does not agree to do not compete or do not solicit clauses._ An NDA should not contain provisions beyond the scope of an NDA. The two most commonly encountered examples of this are the do not compete and do not solicit clauses. We want to be free to hire the best people and make the best products, so when reading through an NDA it is important to keep an eye out for language that prohibits Fleet from hiring or soliciting current or former employees of other companies or that prohibit Fleet from independently developing products that compete with another company's products.  Using the cmd + f function to search for "solici", "compet" and "hir" and reading through the results is a helpful method to quickly scan for these clauses.
+3. Look for any language that discusses a transfer of property rights. Rarely, you may find a clause snuck into an agreement that discusses the transfer of intellectual property rights.  _We want to avoid any situation where Fleet transfers its intellectual property to another party as part of an NDA_.  
+4. Should you find any of the clauses in step 2 or 3, or any other clauses that are beyond the scope of protecting Fleet and the other party's confidential information in a customer NDA or the other party has added these provisions to Fleet's NDA, we want reject this language and communicate that we cannot agree to those terms.
+5. Any concerns or uncertainty over _any_ provisions in an NDA should be brought to Nathanael Holliday in BizOps, who will consult legal counsel if necessary to resolve any concerns.
+
+### Review a vendor agreement
+When reviewing contracts from a vendor, Fleet is concerned about the following:
+1. If there are confidentiality provisions in the agreement, in the place of a stand-alone NDA, we want to make sure that the confidentiality provisions are appropriate and protect Fleet when sensitive data is involved that isn't otherwise available to the public.
```

Comment:
```suggestion
1. If there are confidentiality provisions in the agreement in place of a stand-alone NDA, verify the confidentiality provisions are appropriate and protect Fleet when sensitive data is involved that isn't otherwise available to the public.
```